Update README: Guidance for Customizing Upload Limits

ZarTek-Creole opened the pull request:
This commit adds a new section to the README.md file to provide users with guidance on how to customize upload limits in Lychee Docker container. This update is a follow-up to pull request https://github.com/linuxserver/docker-lychee/pull/64, where it was suggested that instead of changing default settings, it would be more beneficial to provide clear instructions to users on how to make this change themselves. 

The new section includes step-by-step instructions on modifying the `user.ini` file, as well as the necessary commands to restart the Docker container for the changes to take effect. It also adds a note of caution, reminding users that changing these settings may impact their server's performance.

nemchik reviewed (changes requested):
Please note:
https://github.com/linuxserver/docker-lychee/blob/1bb561ce5bc9cb51c19cd3118b12089359ece9a1/README.md?plain=1#L1-L2

You need to edit the `app_setup_block` section here https://github.com/linuxserver/docker-lychee/blob/1bb561ce5bc9cb51c19cd3118b12089359ece9a1/readme-vars.yml#L84

ZarTek-Creole commented:
I apologize for any confusion caused by the previous commits. This commit corrects the application setup instructions in the `readme-vars.yml` file. It provides guidance on customizing Lychee's upload limit to cater to individual server needs. I hope this addresses the concerns raised in the previous discussion, and I appreciate your patience and understanding. Thank you.

Roxedus commented on readme-vars.yml:
```diff
@@ -87,6 +87,35 @@ app_setup_block: |
   Setup mysql/mariadb and account via the webui, accessible at http://SERVERIP:PORT
   More info at [lychee]({{ project_url }}).
 
+  ## 🛠️ Customization
```
```suggestion
  ## Customization
```

Roxedus commented on readme-vars.yml:
```diff
@@ -87,6 +87,35 @@ app_setup_block: |
 
+  ## 🛠️ Customization
+
+  In certain scenarios, you might need to change the default settings of Lychee. For instance, if you encounter limitations when uploading large files, you can increase this limit. 📈
```
```suggestion
  In certain scenarios, you might need to change the default settings of Lychee. For instance, if you encounter limitations when uploading large files, you can increase this limit.
```

Roxedus commented on readme-vars.yml:
```diff
@@ -87,6 +87,35 @@ app_setup_block: |
+
+  In certain scenarios, you might need to change the default settings of Lychee. For instance, if you encounter limitations when uploading large files, you can increase this limit. 📈
+
+  ### 📤 Increasing Upload Limit
```
```suggestion
  ### Increasing Upload Limit
```

Roxedus commented on readme-vars.yml:
```diff
@@ -87,6 +87,35 @@ app_setup_block: |
+
+  After making these changes, you'll need to restart the Docker container for the changes to take effect. Here's how to do it:
+
+  #### 🔁 Via Docker Compose
```
```suggestion
  #### Via Docker Compose
```

Roxedus commented on readme-vars.yml:
````diff
@@ -87,6 +87,35 @@ app_setup_block: |
+  docker-compose restart lychee
+  ```
+
+  #### 🔁 Via Docker Run
````
```suggestion
  #### Via Docker Run
```

Roxedus commented on readme-vars.yml:
````diff
@@ -87,6 +87,35 @@ app_setup_block: |
+  docker restart lychee
+  ```
+
+  ⚠️ Please note that these changes might have implications on your server's performance, depending on its available resources. Thus, it's recommended to modify these settings with caution.
````
```suggestion
  **Please note that these changes might have implications on your server's performance, depending on its available resources. Thus, it's recommended to modify these settings with caution.**
```

Roxedus commented on readme-vars.yml:
```diff
@@ -87,6 +87,35 @@ app_setup_block: |
   Setup mysql/mariadb and account via the webui, accessible at http://SERVERIP:PORT
   More info at [lychee]({{ project_url }}).
 
+  ## Customization
```
```suggestion
  ### Customization
```

Roxedus commented on readme-vars.yml:
```diff
@@ -87,6 +87,35 @@ app_setup_block: |
+
+  In certain scenarios, you might need to change the default settings of Lychee. For instance, if you encounter limitations when uploading large files, you can increase this limit.
+
+  ### Increasing Upload Limit
```
```suggestion
  #### Increasing Upload Limit
```

Roxedus commented on readme-vars.yml:
````diff
@@ -87,6 +87,35 @@ app_setup_block: |
+
+  After making these changes, you'll need to restart the Docker container for the changes to take effect. Here's how to do it:
+
+  #### Via Docker Compose
+
+  ```bash
+  docker-compose restart lychee
+  ```
+
+  #### Via Docker Run
+
+  ```bash
+  docker restart lychee
+  ```
+
````
This should be self-explanatory
```suggestion
```

Roxedus reviewed (changes requested):
Sorry, should have caught the levels of markdown sooner, this all belongs under `Application Setup`, which is at level 2
